packages/boba/register/addresses/addressesArbitrumMainnet_0x.json

```diff
@@ -1,8 +1,8 @@
 {
   "TK_L1ETH": "0x0000000000000000000000000000000000000000",
-  "TK_L2ETH": "0x0000000000000000000000000000000000000000",
+  "TK_L2ETH": "0x4200000000000000000000000000000000000006",
   "Proxy__L1Teleportation": "0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801",
-  "Proxy__L2Teleportation": "0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801",
```

Comment:
Exists: https://arbiscan.io/address/0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801

Reply:
yeah but how about to  boba eth?